rtcclient/client.py

```diff
@@ -912,7 +882,7 @@ def getWorkitem(self, workitem_id, returned_properties=None):
                             proxies=self.proxies,
                             headers=self.headers)
             raw_data = xmltodict.parse(resp.content)
-            workitem_raw = raw_data["oslc_cm:ChangeRequest"]
+            workitem_raw = raw_data["rdf:RDF"]
```

Comment:
This is a breaking change. I'd like to have a way to decide when we should use this new key.

Do you have any wikis on this field change?